lib/src/opentdf.ts

```diff
@@ -759,14 +762,12 @@ class Collection {
     if (opts.ecdsaBindingKeyID) {
       throw new ConfigurationError('custom binding key not implemented');
     }
-    switch (opts.bindingType) {
-      case 'ecdsa':
-        this.encryptOptions = { ecdsaBinding: true };
-        break;
-      case 'gmac':
-        this.encryptOptions = { ecdsaBinding: false };
-        break;
-    }
+
+    // Initialize encryptOptions with policyType if provided
+    this.encryptOptions = {
+      ecdsaBinding: opts.bindingType === 'ecdsa',
```

Comment:
This will always have `encryptOptions.ecdsaBinding` defined, instead of letting it be undefined, which changes the defaulting behavior